README.md

```diff
@@ -82,7 +88,11 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+<adjective> ::= "lazy" | "smelly" | ""
+<noun> ::= "dog" | "cat"
+<verb> ::= "ate" | "ran"
+<adverb> ::= "slowly" | "noisily" | ""
+<sentence_structure> ::= "The" <adjective> <noun> <verb> <adverb> "."
```

Comment:
This is close, but it doesn't allow for multiple adjectives. I was looking for something like

~~~
  <adjectives> ::= "" | <adjectives> <adjective>
  <adjective> ::= "lazy" | "smelly"
~~~

# -2